modules/loader/src/main/scala/com/snowplowanalytics/snowplow/postgres/streaming/source.scala

```diff
@@ -83,36 +88,46 @@ object source {
         val pubsubConfig = PubsubGoogleConsumerConfig[F](onFailedTerminate = pubsubOnFailedTerminate[F])
         PubsubGoogleConsumer
           .subscribeAndAck[F, Either[BadData, Data]](blocker, project, subscription, pubsubErrorHandler[F], pubsubConfig)
+      case Source.LocalFs(path) =>
+        val store: Store[F] = FileStore[F](NioPaths.get("/"), blocker)
+        store.list(path, recursive = true)
+          .map { p =>
```

Comment:
`map + flatten` equals `flatMap`. Although I always prefer `Pipe` and `through` when work with streams.